daemon/execdriver/native/driver.go

```diff
@@ -335,7 +335,9 @@ func (d *driver) Clean(id string) error {
 }
 
 func (d *driver) Stats(id string) (*execdriver.ResourceStats, error) {
+	d.Lock()
 	c := d.activeContainers[id]
+	d.Unlock()
```

Comment:
Ok I can repro the race locally with docker 1.6.2 but with a `-race` build I cannot with this command cause it's to slow and true finishes instantly
```
docker run -d busybox true | xargs docker pause | xargs docker unpause
Error response from daemon: Cannot unpause container 846f61728797250f9119d8a347992cd10c907190177cdb25b4c81d4e4a1a38a9: open /sys/fs/cgroup/freezer/docker/846f61728797250f9119d8a347992cd10c907190177cdb25b4c81d4e4a1a38a9/freezer.state: no such file or directory
FATA[0000] Error: failed to unpause container named 846f61728797250f9119d8a347992cd10c907190177cdb25b4c81d4e4a1a38a9 
```